observations/atmosphere/aircraft.yaml.j2

```diff
@@ -7,146 +7,552 @@
+      is_in: 134
+
+
+  # Assign the initial observation error, based on pressure (for Canadian AMDAR itype=135)
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: assign error
+      error function:
+        name: ObsFunction/ObsErrorModelStepwiseLinear
+        options:
+          xvar:
+            name: MetaData/pressure
+          xvals: [100000, 95000, 90000, 85000, 80000]
+          errors: [1.4706, 1.3529, 1.2353, 1.1176, 1.0 ]
+    where:
+    - variable:
+        name: ObsType/airTemperature
+      is_in: 135
+
+  # Reject where QC = 9-15
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature 
+    where:
+    - variable: PreQC/airTemperature
+      is_in: 9-15
     action:
       name: reject
-    #
-  - filter: Bounds Check
+
+# insert errormod here
+#  # When multiple obs exist within a single vertical model level, inflate ObsError
+#  - filter: Perform Action
+#    filter variables:
+#    - name: airTemperature
+#    action:
+#      name: inflate error
+#      inflation variable:
+#        name: ObsFunction/ObsErrorFactorConventional
+#        options:
+#          test QCflag: PreQC
+#          inflate variables: [airTemperature]
+#          pressure: MetaData/pressure
+#          distance threshold: 60000.0
+#    defer to post: true
+
+  # Error inflation where QC is 3 or 7
+  - filter: Perform Action
     filter variables:
-    - name: windEastward
-    - name: windNorthward
-    test variables:
-    - name: ObsFunction/Velocity
-    maxvalue: 130.0
+    - name: airTemperature 
+    action:
+      name: inflate error
+      inflation factor: 1.2
+    where:
+    - variable: PreQC/airTemperature
+      is_in: 3,7
+
+  # Error inflation when observation pressure < 100 hPa (read_prepbufr.f90)
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature 
+    action:
+      name: inflate error
+      inflation factor: 1.2
+    where:
+    - variable: MetaData/pressure
+      maxvalue: 10000.0
+
+  # Inflate error by 10 if kx==130 and P <500 hPa (read_prepbufr.f90)
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: inflate error
+      inflation factor: 10.0
+    where:
+    - variable: ObsType/airTemperature 
+      is_in: 130
+    - variable: MetaData/pressure
+      minvalue: 50000.0
+
+  # Inflate if IALR > 30
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: inflate error
+      inflation factor: 1.5
+    where:
+    - variable:
+        name: MetaData/instantaneousAltitudeRate 
+      minvalue: 29.9999
+    defer to post: true
+
+  # A temporary solution: Replace error by GsiAdjustObsError overwrite above error assignments and
+  # inflation.
+  - filter: Perform Action
+    filter variables:
+    - name: airTemperature
+    action:
+      name: assign error
+      error function: GsiAdjustObsError/airTemperature
```

Comment:
To do: Need to find a way to avoid dependencies from GSI.

Reply:
Does this need to be done for phase 1?